css-text-4/Overview.bs

```diff
@@ -31,8 +35,519 @@ Introduction</h2>
+
+	<dl dfn-type=value dfn-for=word-boundary-detection>
+		<dt><dfn>manual</dfn>
+		<dd>
+			The User Agent must not insert [=virtual word boundaries=].
+
+			[=Typographic character units=] with class SA in [[!UAX14]]
+			must be treated as if they had class AL
+			(i.e. assuming ''word-break: normal''
+			and a value of 'line-break' other than ''line-break/anywhere'',
+			there is no [=soft wrap opportunity=]
+			between pairs of such characters).
+
+			<div class=advisement>
+				Authors using this value for Southeast Asian languages
+				are expected to manually indicate word boundaries,
+				for instance using <{wbr}> or U+200B.
+				Otherwise, there will be no [=soft wrap opportunity=]
+				and the text may overflow.
+			</div>
+
+		<dt><dfn>normal</dfn>
+		<dd>
+			The User Agent must not insert [=virtual word boundaries=],
+			except within runs of characters belonging to Southeast Asian languages,
+			where content analysis must be performed
+			to determine where to insert [=virtual word boundaries=].
+
+			As with ''word-boundary-detection/manual'',
+			[=typographic character units=] with class SA in [[!UAX14]]
+			must be treated as if they had class AL;
+			however, the User Agent must additionally
+			analyse the content of a run of such characters
+			and insert [=virtual word boundaries=] where appropriate.
+			Within the constraints set by this specification,
+			the specific algorithm used is UA-dependent.
+
+			As various languages can be written in scripts
+			which use the characters with class SA,
+			if the [=content language=] is known,
+			the User Agent should use this information
+			to tailor its analysis.
+
+			In order to avoid overflow,
+			if the User Agent is unable to perform this analysis
+			for any subset of the characters with class SA--
+			for example due to lacking a dictionary for certain languages--
+			there must be a [=soft wrap opportunity=]
+			between pairs of characters in that subset.
+
+			Note: This [=soft wrap opportunity=] is not
+			a [=virtual word boundary=],
+			and is ignored by 'word-boundary-expansion'.
+
+			Note: This provision is not triggered merely when
+			the UA fails to find a word boundary in a particular text run;
+			the text run may well be a single unbreakable word.
+			It applies for example
+			when a text run is composed of Khmer characters (U+1780 to U+17FF)
+			if the User Agent does not know how to determine
+			word boundaries in Khmer.
+
+		<dt><dfn>auto(<<lang>>)</dfn>
+		<dd>
+			This value directs the User Agent to perform language-specific content analysis
+			to determine where to insert [=virtual word boundaries=].
+
+			<dfn dfn-type=type><<lang>></dfn> must be a valid CSS <<ident>> or <<string>>.
+			It represents an IETF BCP 47 language range
+			(see [[BCP47]]).
+			If the UA does not support word-boundary detection
+			for <em>all</em> languages represented by the specified range,
+			it must reject that value at parse-time.
```

Comment:
"rejected at parse-time" is not the right terminology. You want to say it's invalid. E.g.

If the UA does not support word-boundary detection for all languages represented by the specified language range, that specified value is invalid (and will cause the declaration to be ignored).

or

Only `<<lang>>` values for which the UA has support of all languages represented by the specified range are valid.

or somesuch.